HW1/P4/P4.py

```diff
@@ -0,0 +1,47 @@
+from P4_bfs import *
+
+import itertools
+
+# Initialize Spark
+from pyspark import SparkContext
+sc = SparkContext()
+sc.setLogLevel("ERROR")
```

Comment:
Yes - this way Spark only prints out actual errors, as opposed to a running log of everything it does.  Helps de-clutter and focus your attention on the output worth reading.